assets/locales/en/main.json

```diff
@@ -20,7 +20,8 @@
       "change": "Change width of a segment you’re pointing at<br />(hold <kbd class='key'>Shift</kbd> for more precision)"
     },
     "share": {
-      "sign-in": "<a href='/twitter-sign-in?redirectUri=/just-signed-in'>Sign in with Twitter</a> for nicer links to your streets and your personal street gallery",
+      "sign-in-twitter": "<a href='#'>Sign in with Twitter</a>",
```

Comment:
I think we want to remove the anchor text here and just rely on the anchor in JSX

Reply:
Oh okay